src/components/Home/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import Header from '../Header'
+
+export default function Home () {
+  return (
+    <>
+      <Header />
+      <h1>BlackMarket</h1>
```

Comment:
it would be a good idea to use Intl for this project! is something really common, here is the [documentation](https://www.npmjs.com/package/react-intl)